tests/acceptance/bootstrap/CliContext.php

```diff
@@ -431,9 +440,45 @@ public function getJSONDecodedCliMessage(ResponseInterface $response): array {
 	public function cleanUploadsSessions(): void {
 		$command = "storage-users uploads sessions --clean";
 		$body = [
-			"command" => $command
+		  "command" => $command
 		];
 		$response = CliHelper::runCommand($body);
 		Assert::assertEquals("200", $response->getStatusCode(), "Failed to clean upload sessions");
 	}
+
+	/**
+	 * @When the administrator creates folder :folder for user :user
+	 *
+	 * @param string $folder
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function theAdministratorCreatesFolder(string $folder, string $user): void {
+		$userUuid = $this->featureContext->getUserIdByUserName($user);
+		$storagePath = $this->getStoragePath();
+		$body = [
+		  "command" => "mkdir -p $storagePath/$userUuid/$folder",
+		  "raw" => true
+		];
+		$this->featureContext->setResponse(CliHelper::runCommand($body));
+		sleep(1);
```

Comment:
why do we need this sleep?

Reply:
![image](https://github.com/user-attachments/assets/4f9093d3-8a50-4b2a-8609-e21f4078644c)

I think a minimum wait is needed here because the api doesn't return the created resource.

Reply:
hmm could loop checking if the API sees the file, I think that is how we do it for async uploads. Or we keep the sleep for now and wait for https://github.com/opencloud-eu/opencloud/issues/718